darwin/exporter/formats/mask.py

```diff
@@ -62,46 +394,60 @@ def export(annotation_files: Iterable[dt.AnnotationFile], output_dir: Path, mode
         if height is None or width is None:
             raise ValueError(f"Annotation file {annotation_file.filename} references an image with no height or width")
 
-        mask: Image.Image = np.zeros((height, width)).astype(np.uint8)
-        annotations = [a for a in annotation_file.annotations if ispolygon(a.annotation_class)]
-        for a in annotations:
-            if isinstance(a, dt.VideoAnnotation):
-                print(f"Skipping video annotation from file {annotation_file.filename}")
-                continue
+        mask: NDArray = np.zeros((height, width)).astype(np.uint8)
+        annotations: List[dt.AnnotationLike] = [
+            a
+            for a in annotation_file.annotations
+            if a.annotation_class.annotation_type in ["polygon", "complex_polygon", "raster_layer", "mask"]
+        ]
 
-            cat = a.annotation_class.name
-            if a.annotation_class.annotation_type == "polygon":
-                polygon = a.data["path"]
-            elif a.annotation_class.annotation_type == "complex_polygon":
-                polygon = a.data["paths"]
-            sequence = convert_polygons_to_sequences(polygon, height=height, width=width)
-            draw_polygon(mask, sequence, palette[cat])
+        render_type = get_render_mode(annotations)
 
+        if render_type == "raster":
+            # Add categories to list
+            errors, mask, categories, colours = render_raster(
+                mask, colours, categories, annotations, annotation_file, height, width
+            )
+
+        else:
+            #  Add categories to list
+            errors, mask, categories, colours = render_polygons(
+                mask, colours, categories, annotations, annotation_file, height, width
+            )
+
+        if errors:
+            print(f"Errors rendering {annotation_file.filename}:")
+            for e in errors:
+                print(e)
+
+            raise DarwinException.from_multiple_exceptions(errors)
```

Comment:
Not vital here, but we should consider moving the printing logic and console creation inside this, there are multiple places where we follow this printing of multiple exceptions list and would be good to standardize our approach in the future